cypress/e2e/crissy/page-objects/pages/HomePage.js

```diff
@@ -0,0 +1,14 @@
+export default class HomePage {
+
+    //Accessing demoblaze website
+static demoBlazeHome() {
+        cy.visit('https://www.demoblaze.com/index.html#', { timeout: 10000 })
+        cy.url().should('include', 'index.html')
+    }
+    
+//Accessing Cart
+static cart() {
+    cy.get('a').contains('Cart').should('be.visible')
+        .click()
```

Comment:
Often time, we will verify that the button has been correctly clicked by verifying the button is not visible.
This would be best using an alias (and if you are not sure what alias is, you can look it up, but basically, it's a pointer or a reference to an element in the DOM (document object model) = html page.

```
cy.get('a').contains('Cart').as('cartLinkAlias').should('be.visible')
        .click()
cy.get('@cartLinkAlias').should('not.exist');
```
the "as" sets the reference. (you can use alias to record a value as well, anything you'd like)
the get ('@alias') points back to that element and you can chain commands again.
If you were storing a value, you'd use cy.wrap('@alias') to get it back since cy.get is meant for elements on the page.